### PR TITLE
Revert "Update for sslyze changes"

### PR DESCRIPTION
**Pull request by jsf9k**
Reverts cisagov/pshtt#199.  That pull request is not ready to be merged yet.

### CODE REVIEW

**Review by echudow — APPROVED** [review]

**Review by dav3r — APPROVED** [review]
♻️

**Comment by jsf9k** [comment]
FWIW, we (the cisagov devs) usually let the assignee(s) do the actual merging once all the required approvals are in.  Sometimes that person will want to wait for someone who is particularly familiar with a piece of the code to review, even when he or she already has the required two approvals to merge.  We've been thinking of assignees as the folks who shepherd the PR through the process.

FYSA @IanLee1521 and @echudow, since I know you guys have no way to be familiar with our internal processes.

**Comment by IanLee1521** [comment]
Ah, ok. Good to know.

I've often seen work in progress PRs noted as such, often with a subject prefixed with `WIP: ...`

**Pull request merged**
